lib/ui/view/sign_out/sign_out_viewmodel.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/cupertino.dart';
```

Comment:
There should be no material or Cupertino import in the ViewModel

Kindly adjust and fix

Reply:
I've done that. @Blazebrain